examples/kubernetes.json

```diff
@@ -1,34 +1,36 @@
-{
-  "apiVersion": "vlabs",
-  "properties": {
-    "orchestratorProfile": {
-      "orchestratorType": "Kubernetes"
-    },
-    "masterProfile": {
-      "count": 1,
-      "dnsPrefix": "",
-      "vmSize": "Standard_D2_v2"
-    },
-    "agentPoolProfiles": [
-      {
-        "name": "agentpool1",
-        "count": 2,
-        "vmSize": "Standard_D2_v2"
-      }
-    ],
-    "linuxProfile": {
-      "adminUsername": "azureuser",
-      "ssh": {
-        "publicKeys": [
-          {
-            "keyData": ""
-          }
-        ]
-      }
-    },
-    "servicePrincipalProfile": {
-      "clientId": "",
-      "secret": ""
-    }
-  }
-}
\ No newline at end of file
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes"
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "",
+			"vmSize": "Standard_D2_v2",
+			"distro": "aks"
```

Comment:
for now we shouldn't change the default apimodel, we should add a aksbase specific apimodel until we are ready to make it the default

Reply:
good catch, didn't mean to check this in.